python/ray/_raylet.pyx

```diff
@@ -599,6 +595,14 @@ cdef execute_task(
                     raise e
                 if c_return_ids.size() == 1:
                     outputs = (outputs,)
+            if (<int>task_type == <int>TASK_TYPE_ACTOR_CREATION_TASK):
+                # Record actor repr via :actor_name: magic token in the log.
+                # (Phase 2): this covers code run after __init__ finishes.
```

Comment:
```suggestion
                # (Phase 2): after __init__ finishes, we override the log prefix with the full repr() of the
                # actor. The log monitor will scan for the latest magic token.
```

Not actually sure that this is how it works.

Reply:
Done.